rolling/distribution.yaml

```diff
@@ -1539,8 +1539,8 @@ repositories:
       - spinnaker_synchronized_camera_driver
       tags:
         release: release/rolling/{package}/{version}
-      url: https://github.com/ros2-gbp/flir_camera_driver-release.git
-      version: 2.0.15-1
+      url: https://github.com/ros-drivers-gbp/flir_camera_driver-release.git
```

Comment:
This is incorrect; all release repositories in rolling need to be in the `ros2-gbp` organization.